### ISSUE TITLE
Asp.Net Core OAuth Proviver?

### ISSUE THREAD

**Issue by werddomain** [OpenXbox/xbox-webapi-csharp#12]
Do you know if it's possible to implement an Identity Oauth Provider using Xbox Authentication ?

Something similar to [Microsoft.AspNetCore.Authentication.MicrosoftAccount](https://github.com/aspnet/Security/tree/master/src/Microsoft.AspNetCore.Authentication.MicrosoftAccount)

I'm looking to use XBox Live login as the only way to authenticate user on my web app.

Tanks for your api!

**Comment by tuxuser** [comment]
You could wrap this API into your own OAUTH provider but its really not recommended.
This way users have to give away their actual credentials to your webapp :imp:
Please dont go that route!

For proper OAUTH -> https://github.com/OpenXbox/xbox-webapi-python/issues/6

TL;DR; It does not work if you are not a registered ID@Xbox participant.

**Comment by tuxuser** [comment]
Actually I was mistaken, finally figured it out!!!

https://gist.github.com/tuxuser/8b7cc153cdecd0a9c3f2694850fa90bd
